python-regression/tests/features/steps/api_test_steps.py

```diff
@@ -160,20 +161,31 @@ def spam_call(step, api_call, num_tests, node):
 
     # See if call will be made on one node or all
     api_utils.assign_nodes(node, nodes)
-    node = world.config['nodeId']
+    call_node = world.config['nodeId']
```

Comment:
In this one, it's pulling the call node from the precious steps not assigning it, because the input for this call takes in an `all_nodes` argument instead of specifying a specific one. The `call_node` is the node reference that the results of the call will be stored in for reference from any preceding steps that might want to access them.